components/sections/docs/docs-menu.tsx

```diff
@@ -0,0 +1,416 @@
+/** @jsx jsx */
+
+import { jsx, useColorMode } from "theme-ui";
+import Link from "next/link";
+import Collapsible from "react-collapsible";
+import { useState } from "react";
+
+type Props = {
+  selected?: string;
+};
+
+type TriggerProps = {
+  title?: string;
+  selected?: string;
+};
+
+type SectionProps = {
+  title: string;
+  selected: string;
+  links?: {
+    title: string;
+    href: string;
+    links: {
+      title: string;
+      href: string;
+      links: {
+        title: string;
+        href: string;
+      }[];
+    }[];
+  }[];
+};
+
+const sections = [
+  {
+    title: "Video Application Developers",
+    links: [
+      {
+        title: "Overview",
+        href: "docs/video-application-developers/overview",
```

Comment:
Links should have `/` before, so they reference an absolute route. Now they are leading me into a 404